app/stylesheets/config/dimensions.scss

```diff
@@ -20,6 +20,14 @@ $tab-icon-size: 20px;
 
 $footer-height: 100px;
 
+$divider-height: 12px;
+
+$general-content-padding: 30px;
+
+// percentage values
+$full: 100%;
+$half: 50%;
```

Comment:
I think it's ok to hard-code `100%` and `50%` instead of making variables, since the variable name is exactly specific to the value anyway.